src/content/docs/reference/Uniforms/General/playerMood.mdx

```diff
@@ -4,9 +4,10 @@ description: playerMood
 sidebar:
   label: playerMood
   order: 1
-  badge:
-    text: Unfinished
-    variant: caution
 ---
 
-### `uniform float playerMood;`
\ No newline at end of file
+### `uniform float playerMood;`
+
+The 'mood' value for the player. When it hits 1.0, a cave noise is played.
```

Comment:
Although it's great that you linked the wiki, it might be better to add a little more explanation here. Something on the lines of "the value increases ever tick the player is in a dark area", might also be good to mention that it appears in the f3 menu.